components/steps/CheckedIn.tsx

```diff
@@ -89,15 +90,21 @@ const CheckedIn: React.FunctionComponent<Props> = props => {
               <FoxFlex>
                 <Fox fill={HouseFoxColor} width={50} height={50} />
               </FoxFlex>
-              <PointsTitle>You've contributed</PointsTitle>
-              <PointsTitle>{personInfo.totalPoints ?? ""} points!</PointsTitle>
+              <PointsTitle>You have</PointsTitle>
+              <PointsTitle>
+                <b>{personInfo.totalPoints ?? ""} points!</b>
+              </PointsTitle>
```

Comment:
Making default empty is a little weird. Hopefully we won't ever hit this case, but just in case it might be easier to put 0 or something so its less obviously a bug on our end lol

Reply:
going to do this then merge